en/docs/install-and-setup/setup/kubernetes-deployment/kubernetes/am-pattern-2-all-in-one-gw.md

```
@@ -1,634 +1,821 @@
+- Check that tables have been created in each database (the `shared_db` script creates `UM_*` and `REG_*` tables; the `apim_db` script creates `AM_*` tables)
 
-### 1.3 Configure Internal Encryption Key (Mandatory)
+### Step 7 — Create the Keystore Secret { #step-7 }
 
-This section is for the internal encryption key (`wso2.apim.configurations.encryption.key`), which is mandatory and used by API Manager to encrypt and decrypt internal/shared data.
+The Helm chart mounts a Kubernetes secret named `apim-keystore-secret` as a volume into the pods. The pods will not start if this secret does not exist.
 
-1. Generate a unique 256-bit secret key. If you use OpenSSL, the command will be as follows:
+1. Extract the default keystores from your All-in-One image and create the secret:
 
     ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)